docs/src/docs/reports/teststeps.adoc

```diff
@@ -0,0 +1,28 @@
+= Test Steps
+
+It is a good and recommended practice to use test steps to increase the transparency of the test.
+In case of failures it is easier to understand their context:
+
+image::report-ng-02.png[align="center", alt="Test Steps"]
+
+Test steps are defined within the test code by marking their begin as demonstrated in this example test of the testerra skeleton project:
+[source,java]
+----
+    @Test
+    @Fails(ticketString = "http://jira.mms/JIRA-1337", description = "Will fail because user does not exist")
```

Comment:
Please remove the `@Fails` and the `@FailureCorridoe` annotation because its not needed here. It also causes a line break because of character limitations in the code view.

Reply:
ok.